lib/tasks/foreman_ansible_tasks.rake

```diff
@@ -3,10 +3,11 @@
 # Tests
 namespace :test do
   desc 'Foreman Ansible plugin tests'
-  Rake::TestTask.new(:foreman_ansible) do |t|
-    test_dir = File.join(File.dirname(__FILE__), '../..', 'test')
+  Rake::TestTask.new(:foreman_ansible => 'db:test:prepare') do |t|
+    test_dir = File.join(__dir__, '..', '..', 'test')
     t.libs << ['test', test_dir]
     t.pattern = "#{test_dir}/**/*_test.rb"
+    t.test_files = [Rails.root.join('test/unit/foreman/access_permissions_test.rb')]
```

Comment:
There's a related discussion in https://github.com/theforeman/actions/pull/16 on the best way to do this.